*: add tip in contributing.md (#12129)
tikv/tikv: CONTRIBUTING.md

## Running TiKV

To run TiKV as an actual key-value store, you will need to run it as a cluster (a cluster can have just one node, which is useful for testing). You can do this on a single machine or on multiple machines. 

> Install [PD](https://github.com/tikv/pd) to manage the cluster (even if just one node on a single machine). 

Reviewer: May I ask why you change the wording to "install"? "Install" sounds like a strong instruction but we don't actually care about it.

> Install [PD](https://github.com/tikv/pd) to manage the cluster (even if just one node on a single machine). 

Author: ```suggestion
Use [PD](https://github.com/tikv/pd) to manage the cluster (even if just one node on a single machine). 
```

> Instructions are in our [docs](https://tikv.org/docs/dev/tasks/deploy/binary/) (if you build TiKV from source, you  could skip `1. Download package` and `tikv-server` is in directory `/target`).

Reviewer 2: ```suggestion
Instructions are in our [docs](https://tikv.org/docs/dev/tasks/deploy/binary/) (if you build TiKV from source, you could skip `1. Download package` and `tikv-server` is in directory `/target`).
```

> Tips: Run TiKV need `ulimit` >= 82920. WSL2 should notice BerserkerGaruk's solution in [issue](https://github.com/Microsoft/WSL/issues/1688)

Reviewer: ```suggestion
Tips: It's recommended to increase the open file limit above 82920. WSL2 users may refer to [the comment]( https://github.com/Microsoft/WSL/issues/1688#issuecomment-532767317) if having difficulty in changing the `ulimit`.
```

> Tips: It's recommended to increase the open file limit above 82920. WSL2 users may refer to [the comment]( https://github.com/Microsoft/WSL/issues/1688#issuecomment-532767317) if having difficulty in changing the `ulimit`.

Reviewer 2: ```suggestion
Tips: It's recommended to increase the open file limit above 82920. WSL2 users may refer to [the comment](https://github.com/Microsoft/WSL/issues/1688#issuecomment-532767317) if having difficulty in changing the `ulimit`.
```
